app/src/main/res/layout/activity_even_information.xml

```diff
@@ -8,39 +8,43 @@
 
     <EditText
         android:id="@+id/etLocationEven"
-        android:layout_width="320dp"
```

Comment:
Happy to see you're playing around with the UI and customizing the app to your own liking! I think the post-it notes are very cute :)
These changes should be in a different commit & PR, though, since these changes are very distinct from the map changes!